iterator/iterator.py

```diff
@@ -0,0 +1,153 @@
+class Node:
```

Comment:
Your pull request: ITERATOR-1022 Added iterator.py
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']

Your commit: ITERATOR-1022 Added iterator.py
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']
Your commit: ITERATOR-1022 Added class Node
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']
Your commit: ITERATOR-1022 Added method get
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']
Your commit: ITERATOR-1022 Added method addAtHead
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']
Your commit: ITERATOR-1022 Added method addAtIndex
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']
Your commit: ITERATOR-1022 Added method deleteAtIndex
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']
Your commit: ITERATOR-1022 Added methods printlist,next,iter
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']
Your commit: ITERATOR-1022 Added class Student, main
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']
Your commit: ITERATOR-1022 Refactored iterator.py
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']
Action of your comment must be from ['Added', 'Deleted']